### ISSUE TITLE
Text change on collection participants form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#607]
1. Please change the text at the top of the page to an H2: "Collection participants"
2. "Managers" and "Depositors" should be bold paragraph.
